web/src/features/fbaCalculator/validation.ts

```diff
@@ -38,9 +38,22 @@ export const isWindSpeedInvalid = (windSpeed: number | undefined): boolean => {
   return isGreaterThan(windSpeed, 120)
 }
 
+/**
+ * Returns whether precip input is invalid or not
+ * @param row the input row to check against
+ * @returns true if precip is greater than 200 (mm), false otherwise
+ */
+export const isPrecipInvalid = (precip: number | undefined): boolean => {
+  if (_.isUndefined(precip)) {
+    return false
+  }
+  return isGreaterThan(precip, 200)
```

Comment:
Do we also want to check for less than 0?

Reply:
Added a less than 0 check to the validation logic for wind speed and precipitation. It's just a precaution as the input props on the TextField component prevent you from entering a negative number, but allow you enter a number above the input prop max defined value.

Reply:
Done in https://github.com/bcgov/wps/pull/4407/commits/149b8a50e8291b98e6a12fe9f57677c64a451065